From df131a589806aa221a5aa91c30fa15e184117805 Mon Sep 17 00:00:00 2001
From: Ingo Ruhnke <grumbel@gmail.com>
Date: Sun, 15 Jan 2012 20:45:07 +0100
Subject: [PATCH 026/129] Added -framework Cocoa as requested by Josh Dye

---
 SConscript | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/SConscript b/SConscript
index 54b64680d..758567f51 100644
--- a/SConscript
+++ b/SConscript
@@ -149,7 +149,7 @@ class Project:
                                                      'src/engine/display/opengl/opengl_framebuffer.cpp' ],
                                  CPPDEFINES = [('HAVE_OPENGL', 1)])
             if sys.platform == "darwin":
-                self.conf.env.Append(LINKFLAGS = [ '-framework', 'OpenGL' ])
+                self.conf.env.Append(LINKFLAGS = [ '-framework', 'OpenGL', '-framework', 'Cocoa' ])
             else:
                 self.conf.env.Append(LIBS = ['GL'])
 
-- 
2.51.2

