From f1031ba8786bea8b4b700b0ddcebaabc9b1997b5 Mon Sep 17 00:00:00 2001
From: Ingo Ruhnke <grumbel@gmail.com>
Date: Tue, 7 Aug 2012 14:39:37 +0200
Subject: [PATCH 034/129] Added support for the XDG Base Directory
 Specification

---
 src/util/system.cpp | 33 ++++++++++++++++++++++++++++-----
 1 file changed, 28 insertions(+), 5 deletions(-)

diff --git a/src/util/system.cpp b/src/util/system.cpp
index c4e0674ec..99a6f6d2c 100644
--- a/src/util/system.cpp
+++ b/src/util/system.cpp
@@ -317,16 +317,39 @@ System::find_userdir()
 	}
 	
 #else /* !WIN32 */
-  char* homedir = getenv("HOME");
+  // If ~/.pingus/ exist, use that for backward compatibility reasons,
+  // if it does not, use $XDG_CONFIG_HOME, see:
+  // http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
 
-  if (homedir)
+  { // check for ~/.pingus/
+    char* homedir = getenv("HOME");
+    if (homedir && strcmp(homedir, "") != 0)
+    {
+      std::string old_pingus_config_dir = std::string(homedir) + "/.pingus/";
+      if (exist(old_pingus_config_dir))
+      {
+        return old_pingus_config_dir;
+      }
+    }
+  }
+  
+  char* config_dir = getenv("XDG_CONFIG_HOME");
+  if (!config_dir || strcmp(config_dir, "") == 0)
   {
-    return std::string(homedir) + "/.pingus/";
+    char* homedir = getenv("HOME");
+    if (homedir && strcmp(homedir, "") != 0)
+    {
+      return std::string(homedir) + "/.config/pingus/";
+    }
+    else
+    {
+      raise_exception(std::runtime_error, "can't find userdir as neither $HOME nor $XDG_CONFIG_HOME is set");      
+    }
   }
   else
   {
-    raise_exception(std::runtime_error, "Environment variable $HOME not set, fix that and start again.");
-  }
+    return std::string(config_dir) + "/pingus/";
+  }  
 #endif
 }
 
-- 
2.51.2

