From 6c4139bd0d90972bde6caeb1ec2c55823726a9fd Mon Sep 17 00:00:00 2001
From: Ingo Ruhnke <grumbel@gmail.com>
Date: Fri, 10 May 2013 04:23:14 +0200
Subject: [PATCH 045/129] Improved error message on surface load failure

---
 src/engine/display/surface.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/engine/display/surface.cpp b/src/engine/display/surface.cpp
index 5b5123365..8f91884b5 100644
--- a/src/engine/display/surface.cpp
+++ b/src/engine/display/surface.cpp
@@ -69,7 +69,7 @@ Surface::Surface(const Pathname& pathname) :
   SDL_Surface* surface = IMG_Load(pathname.get_sys_path().c_str());
   if (!surface)
   {
-    raise_exception(std::runtime_error, "couldn't load: " << pathname);
+    raise_exception(std::runtime_error, "couldn't load: " << pathname.get_sys_path() << ": " << IMG_GetError());
   }
   else
   {
-- 
2.51.2

