From 90f25d0ff04b9ee9cbc07aacc3042e1a16d6495b Mon Sep 17 00:00:00 2001
From: Ingo Ruhnke <grumbel@gmail.com>
Date: Mon, 23 Mar 2015 17:57:55 +0100
Subject: [PATCH 082/129] Updated tinygettext include path

---
 SConscript | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/SConscript b/SConscript
index c824b3cf3..2cd3b53ee 100644
--- a/SConscript
+++ b/SConscript
@@ -90,7 +90,7 @@ class Project:
         self.opts.Add('CXX', 'C++ Compiler', 'g++')
     #   self.opts.Add('debug', 'Build with debugging options', 0)
     #   self.opts.Add('profile', 'Build with profiling support', 0)
-    
+
         self.opts.Add('CPPPATH',    'Additional preprocessor paths', [])
         self.opts.Add('LIBPATH',    'Additional library paths',      [])
         self.opts.Add('CPPFLAGS',   'Additional preprocessor flags', [])
@@ -111,9 +111,9 @@ class Project:
         self.env.Append(CXXFLAGS = ["-std=c++0x"])
         self.env.Append(CPPDEFINES = [('VERSION', '"\\"%s\\""' % package_version)])
         self.env.Append(CPPPATH = ['src/'])
-        
+
         Help(self.opts.GenerateHelpText(self.env))
-                
+
 
         self.conf = self.env.Configure(custom_tests = {
             'CheckMyProgram' : CheckMyProgram,
@@ -134,7 +134,7 @@ class Project:
             print self.fatal_error
             Exit(1)
 
-    def configure_gxx(self): 
+    def configure_gxx(self):
         # FIXME: Seems to require a rather new version of SCons
         ret = self.conf.CheckBuilder(context, None, "C++")
         if ret != "":
@@ -144,8 +144,8 @@ class Project:
         if not self.env['with_opengl']:
             self.reports += "  * OpenGL support: disabled\n"
         else:
-            self.reports += "  * OpenGL support: enabled\n"           
-            self.conf.env.Append(optional_sources = ['src/engine/display/opengl/opengl_framebuffer_surface_impl.cpp', 
+            self.reports += "  * OpenGL support: enabled\n"
+            self.conf.env.Append(optional_sources = ['src/engine/display/opengl/opengl_framebuffer_surface_impl.cpp',
                                                      'src/engine/display/opengl/opengl_framebuffer.cpp' ],
                                  CPPDEFINES = [('HAVE_OPENGL', 1)])
             if sys.platform == "darwin":
@@ -164,7 +164,7 @@ class Project:
 
     def configure_wiimote(self):
         if not self.env['with_wiimote']:
-            self.reports += "  * Wiimote support: disabled\n"        
+            self.reports += "  * Wiimote support: disabled\n"
         elif self.conf.CheckLibWithHeader('cwiid', 'cwiid.h', 'c++'):
             self.reports += "  * Wiimote support: yes\n"
             self.conf.env.Append(CPPDEFINES = [('HAVE_CWIID', 1)])
@@ -185,7 +185,7 @@ class Project:
             self.conf.env.Append(LIBS = ['Xi'])
             self.conf.env.Append(optional_sources = ['src/engine/input/xinput/xinput_driver.cpp',
                                                      'src/engine/input/xinput/xinput_device.cpp'])
-            
+
     def configure_boost(self):
         if not self.conf.CheckHeader('boost/signals2.hpp', '<>', 'c++'):
             self.fatal_error += "  * library 'boost_signals2' not found\n"
@@ -220,7 +220,7 @@ class Project:
                                      CPPPATH = ['/Library/Frameworks/SDL_mixer.framework/Headers/'])
             else:
                 self.fatal_error += "  * couldn't find sdl-config, SDL missing\n"
-                
+
     def configure_iconv(self):
         iconv_const = self.conf.CheckIconv()
         if iconv_const == False:
@@ -231,11 +231,11 @@ class Project:
 
     def build(self):
         self.env.Append(CPPPATH = ['.', 'src/',
-                                   'external/tinygettext/',
+                                   'external/tinygettext/include/',
                                    'external/logmich/include/'])
 
         libpingus = self.env.StaticLibrary('pingus',
-                                           Glob('external/tinygettext/tinygettext/*.cpp') + \
+                                           Glob('external/tinygettext/src/*.cpp') + \
                                            Glob('external/logmich/src/*.cpp') + \
                                            Glob('src/editor/*.cpp') + \
                                            Glob('src/engine/display/*.cpp') + \
-- 
2.51.2

