From 15c4a7e89f93df114746871aed17214d4f8eee36 Mon Sep 17 00:00:00 2001
From: Ingo Ruhnke <grumbel@gmail.com>
Date: Sun, 27 Oct 2019 21:32:53 +0100
Subject: [PATCH 115/129] Replace sdl-config with pkg-config in SConscript

---
 SConscript | 28 +++-------------------------
 1 file changed, 3 insertions(+), 25 deletions(-)

diff --git a/SConscript b/SConscript
index e521f0eb6..16e18f93d 100644
--- a/SConscript
+++ b/SConscript
@@ -22,26 +22,6 @@ import SCons.Util
 
 Import('package_version')
 
-def CheckSDLLib(context, sdllib):
-    """
-    On some platforms, SDL does this ugly redefine-main thing, that can
-    interact badly with CheckLibWithHeader.
-    """
-    lib = "SDL_%s" % sdllib
-    context.Message('Checking for %s...' % lib)
-    text = """
-#include "SDL.h"
-#include "%s.h"
-int main(int argc, char* argv[]) { return 0; }
-""" % lib
-    context.AppendLIBS(lib)
-    if context.BuildProg(text, ".cpp"):
-        context.Result("failed")
-        return False
-    else:
-        context.Result("ok")
-        return True
-
 def CheckIconv(context):
     context.Message('Check how to call iconv...')
     text = """
@@ -117,7 +97,6 @@ class Project:
 
         self.conf = self.env.Configure(custom_tests = {
             'CheckMyProgram' : CheckMyProgram,
-            'CheckSDLLib': CheckSDLLib,
             'CheckIconv': CheckIconv,
             })
         self.fatal_error = ""
@@ -202,10 +181,9 @@ class Project:
                 self.fatal_error += "  * library 'png' not found\n"
 
     def configure_sdl(self):
-        if self.conf.CheckMyProgram('sdl-config'):
-            self.conf.env.ParseConfig("sdl-config  --cflags --libs | sed 's/-I/-isystem/g'")
-            for sdllib in ['image', 'mixer']:
-                if not self.conf.CheckSDLLib(sdllib):
+        if self.conf.CheckMyProgram('pkg-config'):
+            for sdllib in ['sdl', 'SDL_image', 'SDL_mixer']:
+                if not self.conf.env.ParseConfig("pkg-config  --cflags --libs %s | sed 's/-I/-isystem/g'" % sdllib):
                     self.fatal_error += "  * SDL library '%s' not found\n" % sdllib
         else:
             if sys.platform == 'darwin':
-- 
2.51.2

