--- pingus-0.7.6/Makefile.perms~	2012-06-29 00:45:18.365906013 +0200
+++ pingus-0.7.6/Makefile	2012-06-29 00:45:19.821887810 +0200
@@ -38,7 +38,7 @@ install: install-exec install-data insta
 install-exec: build/pingus
 	install -d "$(DESTDIR)$(BINDIR)"
 
-	install -D build/pingus "$(DESTDIR)$(BINDIR)/pingus.bin"
+	install -m755 -D build/pingus "$(DESTDIR)$(BINDIR)/pingus.bin"
 	echo -e "#!/bin/sh\nexec \"$(BINDIR)/pingus.bin\" --datadir \"$(DATADIR)\"" > "$(DESTDIR)$(BINDIR)/pingus"
 	chmod 755 "$(DESTDIR)$(BINDIR)/pingus"
 
@@ -63,10 +63,10 @@ install-data:
         -name "*.pingus" -o \
         -name "*.levelset" -o \
         -name "*.sprite" \
-        \) -exec install -D {} $(DESTDIR)$(DATADIR)/{} \;
+        \) -exec install -m644 -D {} $(DESTDIR)$(DATADIR)/{} \;
 
 install-man:
-	install -D doc/man/pingus.6 "$(DESTDIR)$(MANDIR)/man1/pingus.6"
+	install -m644 -D doc/man/pingus.6 "$(DESTDIR)$(MANDIR)/man1/pingus.6"
 
 .PHONY : clean install install-exec install-data install-man
 
