--- pingus-0.7.6/src/engine/input/wiimote/wiimote_driver.cpp.wiimote~	2012-06-29 00:30:53.092723416 +0200
+++ pingus-0.7.6/src/engine/input/wiimote/wiimote_driver.cpp	2012-06-29 00:31:15.919438044 +0200
@@ -14,10 +14,12 @@
 //  You should have received a copy of the GNU General Public License
 //  along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
+#include <iostream>
+
 #include "engine/input/wiimote/wiimote_driver.hpp"
 
 #include "engine/input/wiimote/wiimote.hpp"
-#include "pingus/debug.hpp"
+//#include "pingus/debug.hpp"
 #include "pingus/globals.hpp"
 #include "util/string_util.hpp"
 
@@ -48,9 +50,10 @@ WiimoteDriver::update(float delta)
     WiimoteEvent& event = *i;
     if (event.type == WiimoteEvent::WIIMOTE_BUTTON_EVENT)
     {
-      pout(PINGUS_DEBUG_INPUT) << "WiimoteDriver: (wiimote:button (button "
+      /*pout(PINGUS_DEBUG_INPUT) << "WiimoteDriver: (wiimote:button (button "
                                << event.button.button << ")) => "
                                << event.button.down << std::endl;
+      */
                   
       for (std::vector<ButtonBinding>::const_iterator j = button_bindings.begin();
            j != button_bindings.end();
@@ -64,10 +67,11 @@ WiimoteDriver::update(float delta)
     }
     else if (event.type == WiimoteEvent::WIIMOTE_AXIS_EVENT)
     {
-      pout(PINGUS_DEBUG_INPUT) << "WiimoteDriver: (wiimote:axis (axis "
+      /*pout(PINGUS_DEBUG_INPUT) << "WiimoteDriver: (wiimote:axis (axis "
                                << event.axis.axis << ")) => " 
                                << event.axis.pos
                                << std::endl; // Fixme: should output string
+      */
 
       for (std::vector<AxisBinding>::const_iterator j = axis_bindings.begin();
            j != axis_bindings.end(); ++j)
